build-info-extractor-maven3/src/main/java/org/jfrog/build/extractor/maven/BuildInfoRecorder.java

```diff
@@ -729,7 +730,16 @@ private void cleanUpModule() {
     /**
      * Skip the default maven deploy behaviour.
      */
-    private void skipDefaultDeploy(MavenSession session) {
-        session.getUserProperties().put("maven.deploy.skip", Boolean.TRUE.toString());
+    private void skipDeploy(ExecutionEvent event) {
```

Comment:
Let's think about a better name to reflect this method operation. For example: setDeploymentPolicy.
Please also update the Javadoc.